src/components/repo-list.jsx

```diff
@@ -1,15 +1,43 @@
 import React from 'react';
 import RepoCard from './repo-card';
 
-const RepoList = (props) => (
-  <div>
-    <p>Repo List for: {props.username}</p>
-    <ul>
-      <RepoCard username={props.username} />
-      <RepoCard username={props.username} />
-      <RepoCard username={props.username} />
-    </ul>
-  </div>
-);
+const name = 'jackpearsall';
+
+class RepoList extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      data: {},
+    };
+  }
+
+  componentDidMount() {
+    const DATA_URL = `https://mcr-codes-cohorts.herokuapp.com/users/${name}`;
+    fetch(DATA_URL)
+      .then(res => res.json())
+      .then(data => {
+        return this.setState({ data });
+      });
+  }
+
+  render() {
+    const { repos } = this.state.data;
+    return (
+      <div>{ !repos ? <div>loading</div> : repos.map((repo, index) => {
+        return (
+          <RepoCard
+            key={index} 
```

Comment:
using index as key is discouraged, is there nothin in this list that can be used as a unique identifier?

Reply:
```suggestion
            key={repo.name} 
```
We use unique repo names for all our repos